modules/network-observability-total-resource-usage.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+// * network_observability/configuring_operator.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="network-observability-total-resource-usage-table_{context}"]
+= Total average memory and CPU usage
+The following table outlines averages of total resource usage for clusters with a sampling value of 1, 50, and 400.
+
+[IMPORTANT]
+====
+The examples outlined in the table demonstrate scenarios that are tailored to specific workloads. Consider each example only as a baseline from which adjustments can be made to accommodate your workload needs.
+====
+
+.Total average resource usage
+[%autowidth, options="header"]
+|===
+| Sampling value | Parameters | Test 1 (25 nodes) ^[1]^ | Test 2 (65 nodes) ^[2]^ | Test 3 (120 nodes) ^[3]^
+.6+| *Sampling = 1* | *With Loki* 3+|
+| Total NetObserv CPU Usage | 3.24 | 3.42 | 7.30
+| Total NetObserv RSS (Memory) Usage | 14.09 GB | 11.81 GB | 36.46 GB
+| *Without Loki* 3+|
+| Total NetObserv CPU Usage | 2.40 | 2.43 | 5.59
+| Total NetObserv RSS (Memory) Usage | 6.85 GB | 10.39 GB | 13.92 GB
+.6+| *Sampling = 50* | *With Loki* 3+|
+| Total NetObserv CPU Usage | 2.04 | 2.36 | 3.31
+| Total NetObserv RSS (Memory) Usage | 8.79 GB | 19.14 GB | 21.07 GB
+| *Without Loki* 3+|
+| Total NetObserv CPU Usage | 1.55 | 1.64 | 2.70
+| Total NetObserv RSS (Memory) Usage | 6.71 GB | 10.15 GB | 14.82 GB
+.6+| *Sampling = 400* | *With Loki* 3+|
+| Total NetObserv CPU Usage | 1.71 | 1.44 | 2.36
+| Total NetObserv RSS (Memory) Usage | 8.21 GB | 16.02 GB | 17.44 GB
+| *Without Loki* 3+|
+| Total NetObserv CPU Usage | 1.31 | 1.06 | 1.83
+| Total NetObserv RSS (Memory) Usage | 7.01 GB | 10.70 GB | 13.26 GB
```

Comment:
Mehul said one decimal place is fine, so we can go to one.

Reply:
So when Manushri went to round these, there's a couple of times where this creates too similar looking data. For example, *Without Loki*
| Total NetObserv CPU Usage | 2.4 | 2.4 | 5.6

I think for this reason, we should stick to 2 decimal places. 

cc @mpandya15

Reply:
Maybe we could add `total % CPU usage` and `total % memory usage` in seperate rows or inside the same cells ? 

That was a question that often came discussing with customers.

Reply:
Also, should we mention prometheus impact here ? cc @jotak

Reply:
> Maybe we could add `total % CPU usage` and `total % memory usage` in seperate rows or inside the same cells ?
> 
> That was a question that often came discussing with customers.

@jpinsonneau I was chatting w/ @memodi about this, and he says we don't have these metrics now, but it will be something we can capture in future release.

Reply:
Can't we calculate the totals from the nodes hardware specs ? :thinking: 
Then it's a simple rule of three

Reply:
@jpinsonneau I don't know if we want total Netobserv % CPU/RSS Usage as a function of available on nodes OR as a function of cluster usage - from discussions with @stleerh we need latter (which I think makes sense) for which new prometheus queries needs to be added. Also, if we look w.r.t  nodes hardware it would be minuscule usage since that would be dependent on number of nodes in a cluster we obviously have several in each of our tests.

Reply:
For the Prometheus impact, if @jotak agrees, we can add a note like:
[IMPORTANT]
====
Metrics exported to Prometheus can impact the resource usage, so its important to be aware of the cardinality of the values you're exporting. For more information, see Flows format reference. 
====

Then link to the Flows format reference

Reply:
@skrthomas sgtm!

Reply:
@memodi @jpinsonneau It should be with respect to cluster usage.  If it's based on availability, you can allocate a lot of resources without using it so that the percent used by NetObserv is low.